samples/excel/worksheet/worksheet-range-cell.yaml

```diff
@@ -0,0 +1,234 @@
+                let range = sheet.getUsedRange();
+                range.load("address");
+
+                await context.sync();
+
+                console.log(`The address of the used range in the worksheet is "${range.address}"`);
+            });
+
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
+    }
+
+
+    async function getRange() {
+        try {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
+                let range = sheet.getRange();
+                range.load("address");
+
+                await context.sync();
+
+                console.log(`The address of the worksheet range is "${range.address}"`);
+            });
+
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
+    }
+
+
+    async function getCell() {
+        try {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets.getActiveWorksheet();
+                let range = sheet.getCell(1, 4);
+                range.load("address");
+                
+                await context.sync();
+
+                console.log(`The address of the cell with row 2 column 5 is "${range.address}"`);
+            });
+
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
+    }
+
+
+    async function setup() {
+        try {
+            await Excel.run(async (context) => {
+
+                let sheet = await resetWorksheet(context.workbook.worksheets.getActiveWorksheet());
+                
+                let data = [
+                    ["Product", "Qty", "Unit Price", "Total Price"],
+                    ["Almonds", 2, 7.50, "=C3 * D3"],
+                    ["Coffee", 1, 34.50, "=C4 * D4"],
+                    ["Chocolate", 5, 9.56, "=C5 * D5"]
+                ];
+
+                let range = sheet.getRange("B2:E5");
+                range.values = data;
+                range.format.autofitColumns();
+
+                let header = range.getRow(0);
+                header.format.fill.color = "#4472C4";
+                header.format.font.color = "white";
+
+                await context.sync();
+            });
+
+            console.log("Done!");
+        }
+        catch (error) {
+            OfficeHelpers.Utilities.log(error);
+        }
+    }
+
+
+    async function resetWorksheet(sheet: Excel.Worksheet):
+    Promise<Excel.Worksheet> {
+        let context = <Excel.RequestContext>sheet.context;
+
+        sheet.load("name, position");
+
+        await context.sync();
+
+        let newSheet = context.workbook.worksheets.add();
+        newSheet.position = sheet.position;
+        sheet.delete();
+        newSheet.name = sheet.name;
```

Comment:
just to be save I'd re-order those 2 statements (first assign name, then delete).